firefox-ios/Client/Frontend/TrackingProtection/TrackingProtectionHelpers/TrackingProtectionToggleView.swift

```diff
@@ -107,8 +107,14 @@ final class TrackingProtectionToggleView: UIView, ThemeApplicable {
         toggleSwitch.isHidden = isHidden
     }
 
-    func setStatusLabelText(with text: String) {
+    func setStatusLabelText(with text: String, and isAccessibilityTrait: Bool) {
```

Comment:
It seems accessibility traits can be more than just `button`, so maybe we can improve the name here.